distribution/ddf-common/src/main/resources/etc/artemis-backup.xml

```diff
@@ -0,0 +1,171 @@
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ -->
+
+<configuration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+               xmlns="urn:activemq"
+               xsi:schemaLocation="urn:activemq /schema/artemis-configuration.xsd">
+
+    <jms xmlns="urn:activemq:jms">
+        <queue name="DLQ"/>
+        <queue name="ExpiryQueue"/>
+    </jms>
+
+    <core xmlns="urn:activemq:core">
+
+        <persistence-enabled>true</persistence-enabled>
+
+        <!-- this could be ASYNCIO or NIO -->
+        <journal-type>ASYNCIO</journal-type>
+        <paging-directory>./data/artemis/paging</paging-directory>
+        <bindings-directory>./data/artemis/bindings</bindings-directory>
+        <journal-directory>./data/artemis/journal</journal-directory>
+        <large-messages-directory>./data/artemis/large-messages</large-messages-directory>
+
+        <journal-datasync>true</journal-datasync>
+
+        <journal-min-files>2</journal-min-files>
+
+        <journal-pool-files>-1</journal-pool-files>
+
+        <!--
+          You can specify the NIC you want to use to verify if the network
+           <network-check-NIC>theNickName</network-check-NIC>
+          -->
+
+        <!--
+          Use this to use an HTTP server to validate the network
+        <network-check-URL-list>http://www.apache.org</network-check-URL-list> -->
+
+        <network-check-period>10000</network-check-period>
+        <network-check-timeout>255</network-check-timeout>
+
+        <!-- this is a comma separated list, no spaces, just DNS or IPs
+             it should accept IPV6
+
+             Warning: Make sure you understand your network topology as this is meant to validate if your network is valid.
+                      Using IPs that could eventually disappear or be partially visible may defeat the purpose.
+                      You can use a list of multiple IPs, and if any successful ping will make the server OK to continue running -->
+        <network-check-list>${artemis.network.iplist}</network-check-list>
+
+        <!-- use this to customize the ping used for ipv4 addresses -->
+        <!-- <network-check-ping-command>ping -c 1 -t %d %s</network-check-ping-command> -->
```

Comment:
I think you should put a note in the comment about the difference between `ping command` on Windows and *unix (ie on Windows, `ping -n %d %s` works, but the default doesn't)